tests/rptest/scale_tests/high_throughput_test.py

```diff
@@ -0,0 +1,105 @@
+from rptest.services.redpanda import (RESTART_LOG_ALLOW_LIST,
+                                      AdvertisedTierConfig, CloudTierName,
+                                      MetricsEndpoint, SISettings)
+from rptest.tests.prealloc_nodes import PreallocNodesTest
+
+kiB = 1024
+MiB = kiB * kiB
+GiB = kiB * MiB
+
+NoncloudTierConfigs = {
+    #   ingress|          segment size|       partitions max|
+    #             egress|       cloud cache size|connections # limit|
+    #           # of brokers|           partitions min|           memory per broker|
+    "docker-local":
+    AdvertisedTierConfig(3 * MiB, 9 * MiB, 3, 128 * MiB, 20 * GiB, 1, 100, 100,
+                         2 * GiB),
+}
+
+
+class HighThroughputTest2(PreallocNodesTest):
+    def __init__(self, test_ctx: TestContext, *args, **kwargs):
+        self._ctx = test_ctx
+
+        cloud_tier_str = test_ctx.globals.get("cloud_tier", "docker-local")
+        if cloud_tier_str in NoncloudTierConfigs.keys():
+            cloud_tier = None
+            self.tier_config = NoncloudTierConfigs[cloud_tier_str]
+            extra_rp_conf = {
+                'log_segment_size': self.tier_config.segment_size,
+                'cloud_storage_cache_size': self.tier_config.cloud_cache_size,
+                # Disable segment merging: when we create many small segments
+                # to pad out tiered storage metadata, we don't want them to
+                # get merged together.
+                'cloud_storage_enable_segment_merging': False,
+                'disable_batch_cache': True,
+            }
+            num_brokers = self.tier_config.num_brokers
+
+        else:
+            try:
+                cloud_tier = CloudTierName(cloud_tier_str)
+            except ValueError:
+                raise RuntimeError(
+                    f"Unknown cloud tier specified: {cloud_tier_str}. Supported tiers: {CloudTierName.list()+NoncloudTierConfigs.keys()}"
+                )
+            extra_rp_conf = None
+            num_brokers = None
+
+        super(HighThroughputTest2,
+              self).__init__(test_ctx,
+                             1,
+                             *args,
+                             num_brokers=num_brokers,
+                             extra_rp_conf=extra_rp_conf,
+                             cloud_tier=cloud_tier,
+                             disable_cloud_storage_diagnostics=True,
+                             **kwargs)
+
+        if cloud_tier is not None:
+            self.tier_config = self.redpanda.advertised_tier_config
+        test_ctx.logger.info(f"Cloud tier {cloud_tier}: {self.tier_config}")
+
+    @cluster(num_nodes=4, log_allow_list=RESTART_LOG_ALLOW_LIST)
+    def test_throughput_simple(self):
+
+        ingress_rate = self.tier_config.ingress_rate
+        self.topics = [
+            TopicSpec(partition_count=self.tier_config.partitions_min)
+        ]
+        super(HighThroughputTest2, self).setUp()
+
+        msg_size = 16 * kiB
```

Comment:
probably not required for this first test, but an idea for the future would be to have the msg size/msg count be tied to the tiers as this is only 48MiB which probably isn't big enough test for the higher tiers than can do 400MBps ingress

Reply:
Agree, I have tried to fit the test into 30 sec runtime for local DT, great idea to scale it according to the tier